.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Please Add to .gitignore file all of the compiled files.

Example: 
After compiling  index.ts, we get app.js which is not relevant.
- Add to .gitignore: *.js

* scss compiled files aswell.